packages/assets-controllers/src/NftController.test.ts

```diff
@@ -4471,6 +4471,48 @@ describe('NftController', () => {
   });
 
   describe('updateNftMetadata', () => {
+    it('should not update Nft metadata when preferences change and current and incoming state are the same', async () => {
```

Comment:
Nice tests! Is it possible to add a test for the threshold check?

Reply:
It might be a bit tricky to test... Maybe using `Object.defineProperty` to change the threshold amount when testing?

Reply:
This is a nit - can be done later.

Reply:
Thank you @Prithpal-Sooriya  I have tried using the `Object.defineProperty` which didn't do the trick; The error i had was: `TypeError: Cannot redefine property:`
I can address later 🙏

Reply:
Yep can be addressed later.